custom/icds_reports/reports/awc_reports.py

```diff
@@ -1236,7 +1237,7 @@ def get_pregnant_details(case_id, awc_id):
     'start', 'length', 'order', 'reversed_order', 'awc_id'
 ], timeout=30 * 60)
 def get_awc_report_lactating(start, length, order, reversed_order, awc_id):
-    latest_available_month = datetime.utcnow() - timedelta(days=1)
+    latest_available_month = date.today() - timedelta(days=1)
```

Comment:
for my own education, is this fixing anything or just being more consistent with other reports?

Reply:
Just consistency, and because we dont actually need the time